src/react/AppProvider.jsx

```diff
@@ -62,7 +63,7 @@ export default function AppProvider({ store, children }) {
       >
         <IntlProvider locale={locale} messages={getMessages()}>
           <OptionalReduxProvider store={store}>
-            <Router history={history}>
+            <Router history={history} basename={getConfig().BASENAME}>
```

Comment:
We will have to use this, once we migrate to React-router v6

https://github.com/ReactTraining/react-router/blob/dev/docs/advanced-guides/migrating-5-to-6.md#move-basename-from-router-to-routes